index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@0;1&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Sign Up</title>
+</head>
+
+<body>
+
+  <div class="singin-button">
+    <div class="signin-info">
+      <p>Already have an Intuit account?</p>
+      <a href="">I forgot my userID or Password</a>
+    </div>
+    <div>
+      <button class="sign-in-button"><img class="signin-logo" src="img/signinlogo.png" alt=""> Sign In</button>
```

Comment:
- [ ] the button is not click-able please make the changes so that it looks exactly like on the [original page](https://accounts.intuit.com/signup.html).